CODEOWNERS

```diff
@@ -4,32 +4,55 @@
 
 distribution/ddf-common/*	@stustison
 
+distribution/docs/*         @ricklarsen
+
 
 ###############################
 # CATALOG
 ###############################
 
+catalog/async/*   @peterhuffer  @emmberk
+
+catalog/core/catalog-core-api/*             @brendan-hofmann
```

Comment:
I thought it was supposed to be `catalog/core/catalog-core-api/data/types`, or do you need the `src/main/java/ddf/catalog` part as well? Regardless, I was only trying to sign up for the taxonomy, not our entire API. 😺